samples/zigbee/ncp/sample.yaml

```diff
@@ -5,28 +5,30 @@ sample:
 tests:
   zigbee.ncp:
     build_only: true
-    platform_allow: nrf52833dk_nrf52833 nrf52840dk_nrf52840 nrf21540dk_nrf52840
-    tags: ci_build
+    platform_allow: nrf52833dk_nrf52833 nrf52840dk_nrf52840 nrf21540dk_nrf52840 nrf5340dk_nrf5340_cpuapp
+    tags: ci_build ncp
     integration_platforms:
       - nrf52840dk_nrf52840
       - nrf52833dk_nrf52833
       - nrf21540dk_nrf52840
+      - nrf5340dk_nrf5340_cpuapp
 
   zigbee.ncp.usb:
     build_only: true
-    platform_allow: nrf52833dk_nrf52833 nrf52840dk_nrf52840 nrf52840dongle_nrf52840 nrf21540dk_nrf52840
-    tags: ci_build
-    extra_args: CONF_FILE=prj_usb.conf
+    platform_allow: nrf52833dk_nrf52833 nrf52840dk_nrf52840 nrf52840dongle_nrf52840 nrf21540dk_nrf52840 nrf5340dk_nrf5340_cpuapp
+    tags: ci_build ncp
+    extra_args: OVERLAY_CONFIG=overlay-via-usb.conf
```

Comment:
@sebastiandraus do you remember why that was a `CONF_FILE` instead of `OVERLAY_CONFIG`?
Was it about adding several overlays (e.g. for enabling building NCP FW from sources)?

Reply:
Back to prj_usb.conf. Previous propose solution with overlay wasn't correct